shell/initialize/entry-helpers.js

```diff
@@ -195,26 +194,6 @@ async function render(to, from, next) {
   const matches = [];
   const Components = getMatchedComponents(to, matches);
 
-  // If no Components matched, generate 404
-  if (!Components.length) {
```

Comment:
This will no longer ever be true because our 404 page will match if nothing else will.